src/react-components/media-tiles.js

```diff
@@ -243,7 +243,20 @@ class MediaTiles extends Component {
                 <FontAwesomeIcon icon={faPencilAlt} />
               </a>
             )}
+          {entry.type === "room" && (
+            <div className={styles.info}>
+              <FontAwesomeIcon icon={faUsers} />
+              <span>{entry.participant_count}</span>
```

Comment:
probably worth the `|| 0` for backwards compatibility, with old reticulum servers. not usually something to worry about (only affects up-to-date forked clients running old ret servers), but it's so trivial here seems worthwhile.